.github/workflows/update-docs-in-website.yml

```diff
@@ -0,0 +1,121 @@
+      - "**/*.md"
+  workflow_dispatch:
+
+jobs:
+  Make-PR:
+    name: Make PR on website repository with updated latest Bindings documentation
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    steps:
+      - name: Checkout Current repository
+        uses: actions/checkout@v3
+        with:
+          path: bindings
+      - name: Checkout Another repository
+        uses: actions/checkout@v3
+        with:
+          repository: asyncapi/website
+          path: website
+          token: ${{ env.GITHUB_TOKEN }}
+      - name: Config git
+        run: |
+          git config --global user.name asyncapi-bot
+          git config --global user.email info@asyncapi.io
+      - name: Create branch
+        working-directory: ./website
+        run: |
+          git checkout -b update-bindings-docs-${{ github.sha }}
+      - name: Update edit-page-config.json
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const { writeFile } = require('fs').promises;
+            const configPath = './website/config/edit-page-config.json';
+            const checkSlug = 'reference/bindings/';
+            const slug = {
+              "value": checkSlug,
+              "href": "https://github.com/asyncapi/bindings/tree/master"
+            };
+
+            const configData = require(configPath);
+            const entryExists = configData.some(entry => entry.value === checkSlug);
+            if (!entryExists) {
+              configData.unshift(slug);
+              await writeFile(configPath, JSON.stringify(configData, null, 2))
+            }
+
+      - name: Update title and weight of the markdown files
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const fs = require('fs');
+            const path = require('path');
+            const rootPath = './bindings/';
+            let itemIndex = 10;
+
+            function processMarkdownFiles(folderPath, isRoot = true) {
+              const items = fs.readdirSync(folderPath, { withFileTypes: true });
+              for (const item of items) {
+                const fullPath = path.join(folderPath, item.name);
+                if (item.isDirectory()) {
+                  // Always process subdirectories, mark isRoot as false for recursive calls
+                  processMarkdownFiles(fullPath, false);
+                } else if (item.name.endsWith('.md') && !isRoot) { // Skip root level .md files
+                  const baseName = path.basename(fullPath, '.md');
+                  const parentDirName = path.basename(folderPath);
+                  const newFileName = `${parentDirName}.md`;
+                  const newFullPath = path.join(folderPath, newFileName);
+                  fs.renameSync(fullPath, newFullPath);
+                  
+                  const newData = `---\ntitle: '${parentDirName.charAt(0).toUpperCase() + parentDirName.slice(1)}'\nweight: ${itemIndex}\n---\n\n`;
```

Comment:
no upper case please.

let's keep oryginal name as it is used inside AsyncAPI documents otherwise we will have problems as some names should be with upper case, some with all caps

so please stick to oryginal all lower case